src/test/java/core/basesyntax/service/ParserServiceImplTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ParserServiceImplTest {
+    private static final String INPUT_FILE_PATH = "src/test/resources/input.csv";
+    private ReaderServiceImpl readerService;
+    private ParserServiceImpl parserService;
+    private FruitTransaction fruitTransaction;
+
+    @Before
+    public void setUp() {
+        readerService = new ReaderServiceImpl();
+        parserService = new ParserServiceImpl();
+        fruitTransaction = new FruitTransaction(FruitTransaction.Operation.BALANCE,"banana",20);
+    }
+
+    @Test
+    public void parserService_parseFile_Ok() {
```

Comment:
Fix all method names in the project.
![image](https://user-images.githubusercontent.com/20326858/227768201-f30a07c4-c038-4f3f-80b2-9df0c8f56583.png)